### PR TITLE
QOLDEV-1006 remove organisation redirects that should be handled by PURL

### PR DESCRIPTION
**Pull request by ThrawnCA**
Hard-coding redirects in Nginx is much less flexible.

### CODE REVIEW

File: templates/default/nginx.conf.erb [hunk 1]
```diff
@@ -64,16 +64,6 @@
             # redirect 'group' URLs to organisations
             rewrite ^/group/(aboriginal-and-torres-strait-islander-and-multicultural-affairs|agriculture-fisheries-and-forestry|communities-child-safety-and-disability-services|crime-and-misconduct-commission-queensland|education-training-and-employment|energy-and-water-supply|environment-and-heritage-protection|housing-and-public-works|justice-and-attorney-general|local-government-community-recovery-and-resilience|national-parks-recreation-sport-and-racing|natural-resources-and-mines|police|premier-and-cabinet|public-safety-business-agency|queensland-fire-and-emergency-services|queensland-health|science-information-technology-innovation-and-the-arts|state-development-infrastructure-and-planning|tourism-major-events-small-business-and-the-commonwealth-games|transport-and-main-roads|treasury-and-trade) /organization/$1 permanent;
```
Comment (chris-randall-qol) [inline templates/default/nginx.conf.erb:65]: Should we also include this line? It has hardcoded department names also

Comment (duttonw) [inline templates/default/nginx.conf.erb:65]: should this also be a purl ruling?

**Review by duttonw — APPROVED** [review]

File: templates/default/nginx.conf.erb (hunk 1 above)
Comment (ThrawnCA) [reply templates/default/nginx.conf.erb:65]: It's possible. It would potentially be a lot of rules.

Comment (ThrawnCA) [reply templates/default/nginx.conf.erb]: Dropped as PURL _can_ handle it if needed.

**Review by duttonw — APPROVED** [review]

**Pull request merged**